test/smoke/dapi.js

```diff
@@ -1,10 +1,52 @@
+const { client: jaysonClient } = require('jayson/promise');
+const RpcClient = require('@dashevo/dashd-rpc/promise');
+const getNetworkConfig = require('../../lib/test/getNetworkConfig');
+
+const networkConfig = getNetworkConfig();
+
 describe('DAPI', () => {
-  // getBlockHash height:1
-  it('should respond data from chain');
+  networkConfig.inventory.masternodes.hosts.forEach((nodeName) => {
+    describe(nodeName, () => {
+      const dapiClient = jaysonClient.http({
+        host: networkConfig.inventory._meta.hostvars[nodeName].public_ip,
+        port: 3000,
+      });
+      const config = {
+        protocol: 'http',
+        user: 'dashrpc',
+        pass: 'password',
+        host: networkConfig.inventory._meta.hostvars[nodeName].public_ip,
+        port: 20002,
+      };
+      const rpc = new RpcClient(config);
+      // getBlockHash height:1
+      it('should respond data from chain', async () => {
+        const { result } = await dapiClient.request('getBlockHash', { height: 1 });
+        const blockHashRpc = await rpc.getBlockHash(1);
+        expect(result).to.be.equal(blockHashRpc.result);
+      });
+
+      // getBestBlockHeight
+      it('should respond data from insight', async () => {
+        const { result } = await dapiClient.request('getBestBlockHeight', { blockHeight: 1 });
+        expect(result).to.be.an('number');
+        expect(result % 1).to.be.equal(0);
+      });
 
-  // getBestBlockHeight
-  it('should respond data from insight');
+      // getBalance from dapi
+      it('should respond getBalance from drive', async () => {
+        const { result } = await dapiClient.request('getBalance',
+          { address: `${networkConfig.variables.faucet_address}` });
+        expect(result).to.be.an('number');
+        expect(result % 1).to.be.equal(0);
+      });
 
-  // fetchDapObjects any dapiId and type
-  it('should respond data from drive');
+      // fetchDapContract any dapiId and type
+      xit('should respond fetchDapContract', async () => {
```

Comment:
should respond data from drive

Reply:
still doesn't work?

Reply:
{ jsonrpc: '2.0',
  id: 'fc886477-375b-4752-b14c-164143bb1b5f',
  error: { code: 100, message: 'Initial sync in progress' } }